index.html

```diff
@@ -45,6 +45,155 @@ <h2 class="Introduction">I am a Software Developer</h2>
         don't hesitate to contact me.</p>
 </section>
 </div>
+</section>
+<section class="recent-works">
+    <div class="title">
+        <h1>My Recent Works</h1>
+        <hr class="small-line">
+    </div>
+    <section class="work-section">
+        <div class="story">
+            <img src="images/Img Placeholder.png" alt="portfolio-image">
+            <div class="description-block">
+                <h1>Multi-Post Stories</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <nav class="list">
+                    <ul class="nav-list">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </nav>
+                <button type="button">See Project</button>
+            </div>
```

Comment:
- [ ] I noticed that you have a second (`section` tag) here. I would suggest you remove that one and enclose each work card in a (`div` tag) so you can properly style the contents of each work with flexbox. :100: 
- [ ] Currently, you have no styles applied to all your work cards and this makes them incomplete. I also noticed that you only have one work image present. Please add all the other images as seen in the Figma design and add the necessary styles :100: 
- [ ] I highly advise that you stick to the Figma design guidelines as much as possible and this includes fonts, colors, margins, images, etc. in order to build your project properly and in order to get approval :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/213860639-79f812ef-6679-426f-90d9-25eef41bb461.png)
![image](https://user-images.githubusercontent.com/102699958/213860663-8b646558-22f2-48ff-8397-f893db048904.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213860720-f6b4c61f-51f8-4265-8283-e6247bcdaa8e.png)